Allow int and string (composite) primary keys

dkaera opened the pull request:
Closes #164

vitusortner commented:
@dkaera  Thank you for your contribution! I've just added test for the new functionality and changed the naming a bit.

dkaera commented:
> @dkaera Thank you for your contribution! I've just added test for the new functionality and changed the naming a bit.

You're welcome. I'm glad to help.
